src/main/java/org/opensearch/knn/index/query/KNNQueryBuilder.java

```diff
@@ -128,6 +216,12 @@ public KNNQueryBuilder(StreamInput in) throws IOException {
             if (isClusterOnOrAfterMinRequiredVersion("ignore_unmapped")) {
                 ignoreUnmapped = in.readOptionalBoolean();
             }
+            if (isClusterOnOrAfterMinRequiredVersion(KNNConstants.RADIAL_SEARCH_KEY)) {
+                max_distance = in.readOptionalFloat();
+            }
+            if (isClusterOnOrAfterMinRequiredVersion(KNNConstants.RADIAL_SEARCH_KEY)) {
+                min_score = in.readOptionalFloat();
+            }
```

Comment:
nit: we can have both reads under case if case. not a blocker, can be done in a refactoring PR in future

Reply:
Sure, we can do it in future PR